### PR TITLE
Update fido quick start

### PR DESCRIPTION
**Pull request by AfraHussaindeen**
### Purpose
$Subject

### Related Issues
- https://github.com/wso2/product-is/issues/17544

https://github.com/wso2/identity-apps/assets/47135592/0517769f-a9aa-44eb-9334-5c17523de46b

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by pavinduLakshan — DISMISSED** [review]

File: apps/console/src/extensions/components/authenticators/fido/quick-start.tsx [hunk 1]
```diff
@@ -120,6 +123,119 @@ const FIDOQuickStart: FunctionComponent<FIDOQuickStartPropsInterface> = (
...
+                                        <span>
+                                            <Trans
+                                                i18nKey={ "extensions:develop.identityProviders.fido." +
+                                                "quickStart.steps.configureParameters.content.parameters." +
+                                                "progressiveEnrollment.note" }
+                                            >
+                                                If Passkey is set as a first factor, the following 
+                                                <strong> adaptive script</strong> should be added under 
+                                                the <strong>Sign-In-Method</strong> tab of the application. 
+                                                This script is used to verify the user&apos;s identity
```
Comment (himeshsiriwardana) [inline apps/console/src/extensions/components/authenticators/fido/quick-start.tsx:196]: This script is added automatically with the template-based passkey setup and is used to verify the user's identity before enrolling passkeys. However, if you're configuring passkeys without a template, remember to add the script manually.

Comment (AfraHussaindeen) [reply apps/console/src/extensions/components/authenticators/fido/quick-start.tsx]: Resolved via https://github.com/wso2/identity-apps/pull/4750/commits/63c0c3ad085943ac55238c33cf3756343f7c88d1

File: apps/console/src/extensions/i18n/resources/en-US/extensions.ts [hunk 2]
```diff
@@ -2681,6 +2681,38 @@ export const extensions: Extensions = {
...
+                                        "the <3>Sign-In-Method</3> tab of the application. " +
+                                        "This script is added automatically with the template-based " + 
+                                        "Passkey setup and is used to verify the user's " +
+                                        "identity before enrolling passkeys. " +
+                                        "However, if you're configuring Passkey " +
+                                        "without a template, remember to add the script manually."
+                                    },
+                                    usernamelessAuthentication: {
+                                        description: "Enabling this feature allows users to log in with a passkey " +
+                                        "without entering their username, creating a more streamlined " +
```
Comment (himeshsiriwardana) [inline apps/console/src/extensions/i18n/resources/en-US/extensions.ts:2704]:
```suggestion
                                        "without entering a username, creating a more streamlined " +
```

File: apps/console/src/extensions/components/authenticators/fido/quick-start.tsx [hunk 3]
```diff
@@ -120,6 +123,118 @@ const FIDOQuickStart: FunctionComponent<FIDOQuickStartPropsInterface> = (
...
+                                                "quickStart.steps.configureParameters.content.parameters." +
+                                                "progressiveEnrollment.note" }
+                                            >
+                                                If Passkey is set as a first factor, the following 
+                                                <strong> adaptive script</strong> should be added under 
+                                                the <strong>Sign-In-Method</strong> tab of the application.
+                                                This script is added automatically with the template-based 
+                                                Passkey setup and is used to verify the user&apos;s identity 
+                                                before enrolling passkeys. However, if you&apos;re configuring 
+                                                Passkey without a template, remember to add the script manually.
```
[the remark this answers is not in the document]
Comment (AfraHussaindeen) [reply apps/console/src/extensions/components/authenticators/fido/quick-start.tsx:199]: Here the `Passkey` refers the name of the authenticator. Hence it should be in singular form

File: apps/console/src/extensions/i18n/resources/en-US/extensions.ts (hunk 2 above)
Comment (AfraHussaindeen) [reply apps/console/src/extensions/i18n/resources/en-US/extensions.ts]: Resolved via https://github.com/wso2/identity-apps/pull/4750#:~:text=Fix%20review%20comments-,75decbc,-Merge%20state

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**